cli/dts/lib.deno.ns.d.ts

```diff
@@ -66,27 +66,62 @@ declare interface PerformanceMeasureOptions {
 
 declare namespace Deno {
   /** A set of error constructors that are raised by Deno APIs. */
-  export const errors: {
-    NotFound: ErrorConstructor;
-    PermissionDenied: ErrorConstructor;
-    ConnectionRefused: ErrorConstructor;
-    ConnectionReset: ErrorConstructor;
-    ConnectionAborted: ErrorConstructor;
-    NotConnected: ErrorConstructor;
-    AddrInUse: ErrorConstructor;
-    AddrNotAvailable: ErrorConstructor;
-    BrokenPipe: ErrorConstructor;
-    AlreadyExists: ErrorConstructor;
-    InvalidData: ErrorConstructor;
-    TimedOut: ErrorConstructor;
-    Interrupted: ErrorConstructor;
-    WriteZero: ErrorConstructor;
-    UnexpectedEof: ErrorConstructor;
-    BadResource: ErrorConstructor;
-    Http: ErrorConstructor;
-    Busy: ErrorConstructor;
-    NotSupported: ErrorConstructor;
-  };
+  export namespace errors {
+    export class NotFound extends Error {
+      name: "NotFound";
+    }
+    export class PermissionDenied extends Error {
+      name: "PermissionDenied";
+    }
+    export class ConnectionRefused extends Error {
+      name: "ConnectionRefused";
+    }
+    export class ConnectionReset extends Error {
+      name: "ConnectionReset";
+    }
+    export class ConnectionAborted extends Error {
+      name: "ConnectionAborted";
+    }
+    export class NotConnected extends Error {
+      name: "NotConnected";
+    }
+    export class AddrInUse extends Error {
+      name: "AddrInUse";
+    }
+    export class AddrNotAvailable extends Error {
+      name: "AddrNotAvailable";
+    }
+    export class BrokenPipe extends Error {
+      name: "BrokenPipe";
+    }
+    export class AlreadyExists extends Error {
+      name: "AlreadyExists";
+    }
+    export class InvalidData extends Error {
+      name: "InvalidData";
+    }
+    export class TimedOut extends Error {
+      name: "TimedOut";
+    }
+    export class Interrupted extends Error {
+      name: "Interrupted";
+    }
+    export class WriteZero extends Error {
+      name: "WriteZero";
+    }
+    export class UnexpectedEof extends Error {
+      name: "UnexpectedEof";
```

Comment:
``` typescript
name: "UnexpectedEof" | "PartialReadError";
```
This fixes the error but may introduce future issues with `name` typing. Since is expected to have just one value.

Reply:
It's not really that important for `name` to be statically typed, so the simplest way to deal with it would probably just be to change it back to `string`.

Reply:
Or to not support versions below std@0.92.0 🤭 but got your point, thank you!!!